### ISSUE TITLE
Omelette command creates broken symlink inside volto

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#3281]
Tested with @sneridagh , this should be the correct script in package.json:

```
"omelette": "if [ ! -d omelette ]; then ln -sf node_modules/@plone/volto omelette; fi",
```

The generators have to be updated as well.

**Comment by tiberiuichim** [comment]
This causes warnings like:

```
[Error: ENOENT: no such file or directory, stat '/home/tibi/work/rancher/semsearch/node_modules/@plone/volto/volto']
```

**Comment by davisagli** [comment]
Fixed in #3284
